app/models/authorization.rb

```diff
@@ -41,9 +41,8 @@ def authorized?(account)
   end
 
   def cache_key
-    token_keys = (token.attributes || {}).keys.sort.map { |k| "#{k}:#{token.attributes[k]}" }
     key_components = ['c', self.class.model_name.cache_key]
-    key_components << OpenSSL::Digest::MD5.hexdigest(token_keys.join(','))
+    key_components << OpenSSL::Digest::MD5.hexdigest(token.resources.join(' '))
```

Comment:
Oh right ... CMS attempts to cache authorized resources.  Hmmm... I suppose this should still work.